lib/pages/users/csv_downloads.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Users
+  # page object for CSVs
+  class Csv
+    include RSpec::Matchers
+    include Capybara::DSL
+
+    def check_labels(report:, expected_labels:)
+      first('.pjax', text: report).click
+      click_on 'Export found ' + report
```

Comment:
@a-ashbeck up to you if you want to change but it may be easier to read/understand if you interpolate `click_on "Export found #{report}"`